### PR TITLE
add information about meeting times/location

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ You can reach the maintainers of this project at:
 
 - [Slack channel](https://kubernetes.slack.com/messages/cluster-addons)
```
Comment (timothysc) [inline README.md]: imo I would just point to the community repo as the canonical source of truth.

Comment (dholbach) [reply README.md]: Hrm. Where? I don't find any of these useful for anyone: https://github.com/kubernetes/community/search?q=cluster+addons+slack&unscoped_q=cluster+addons+slack

Comment (timothysc) [reply README.md]: You should be able to add all the deets https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle and just link

**Comment by dholbach** [comment]
Pointing to sig-cluster-lifecycle README. As soon as https://github.com/kubernetes/community/pull/3698 is landed, all the info should be there.

**Pull request merged**